README.md

```diff
@@ -92,13 +92,12 @@ Merges to the `master` branch will be published as a prerelease. Prereleases wil
 The following circumstances will be considered a MAJOR or BREAKING change:
 * Droppping existing classnames, css variables, color names, color groups, spacing parameters
 * The existing underlying HTML markup of a component is altered
+* Non-visual HTML attribute changes/additions (such as `role`, `aria-*`, `data-*`)
```

Comment:
perhaps an additional clarification sentence above, something like 

"Fundamental Styles provides CSS directly, and HTML as reference to consumers. Because of thes reference relationship of the HTML seen in fundamental styles, we want to be very clear when we alter that reference so that it is properly reflected in JS implementation libraries. Because of this, even non-visual changes will be treated as breaking."

Reply:
@jbadan I think that what @bcullman wrote is correct, could you add this as well?

Reply:
@bcullman @rengare I also removed the note about altering unit tests.... because we don't have them here 😆 I think that line was more tailored to the implementation libraries.